training/culture-and-process/CONTRIBUTING_TO_20_TIME_BACKLOG.md

```diff
@@ -0,0 +1,54 @@
+Contributing to 20% Backlog in DP
+=================================
+
+There are many ways in which the Trello 20% backlog board gets populated and then prioritised by the Technical Leads.
+
+Below is how you can help to get your ideas and resolving bugbears you may have onto the 20% time board to fix or improve our way of working in development.
+
+### Pre-reading
+
+- It is useful to have an understanding of the [Trello board and flow of stories](./TRELLO_BOARD_FLOW.md) and how we use it to organise development workflow as it is mentioned in this documentation
+- It is useful to have an understanding of [70%, 20% and 10% time](./70_20_10_TIME.md)
+
+### How to get ideas onto the 20% Trello Board?
+
+It is beneficial to yourself and others to bounce off ideas that you may think would help the development team before progressing an idea. This can really help develop an idea into something more tangible and allow a larger buy in from the wider team. It could also identify why this might not be a good idea, or there is not a problem to solve here, either way it is still encouraged to voice any ideas and not be disheartened by any push back, remember we are one team and consensus from the team is important.
+
+- It is advisable to reach out to your peers to discuss any ideas, this can be done in any guilds you are part of.
+- If unsure one can raise with a (S7) Senior Developer or Technical Lead.
+
+Once you are confident in your idea/suggestion, you may add your story to [triage column in the 20% backlog board](https://trello.com/b/5G8rf9cm/20-time-backlog).
+
+### What should be included in the 20% story?
+
+Please remember that when a 20% story is triaged and looked at by others that they will need to be able to understand the story so they can clearly make a decision on the value of this story and prioritise it according to the existing backlog of 20% work.
+
+One should use the relevant templates available on the trello board, these can be found in the column at the end of the board by scrolling right.
```

Comment:
the column on the right is good for getting familiar with the templates, but you can use the 'create from template...' button on the bottom right of a column when creating stories. Might be worth explaining both? i know we might end up explaining that in a few places, but its probably not a lot of extra detail for the ease of access it might provide

Reply:
I didn't know this, so added descriptions on how to copy from templates (prev. i just used the copy button) - also added how to create template card